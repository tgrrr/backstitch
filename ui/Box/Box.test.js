import React from 'react'
import renderer from 'react-test-renderer'
import { Box } from '.'

it('Box: default', () => {
    const component = renderer.create(
        <Box>
            <>
                Box
            </>
        </Box>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
