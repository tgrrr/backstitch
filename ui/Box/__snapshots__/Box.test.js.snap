// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Box "
    data-testid="Box"
  >
    <div
      className="MuiBox-root MuiBox-root-1"
    >
      Box
    </div>
  </div>
</div>
`;
